packages/cli/src/utils/installDeps.ts

```diff
@@ -22,6 +22,7 @@ export const installDeps = async (
     sp.start(picocolors.cyan(entryMessage));
 
     await execa(packageManager, ["install"], options);
+    await execa(packageManager, ["install"], { cwd: `${options.cwd}/cli` });
```

Comment:
Two installs?

Reply:
yes, the cli is not part of the main packages